services/web-server/src/resolvers/Hooks.js

```diff
@@ -40,6 +45,9 @@ export default {
     hookStatus(parent, { hookGroupId, hookId }, { loaders }) {
       return loaders.hookStatus.load({ hookGroupId, hookId });
     },
+    hookLastFires(parent, {hookGroupId, hookId, filter}, {loaders}) {
+      return loaders.hookLastFires.load({hookGroupId, hookId, filter});
```

Comment:
Add a space after `{` and before `}` when destructuring:

`return loaders.hookLastFires.load({ hookGroupId, hookId, filter });`